account/tests.py

```diff
@@ -1,40 +1,71 @@
 import unittest
 import re
-from account.models import validate_gov_id, validate_phone, validate_pin_code
 from django.core.exceptions import ValidationError
+from account.factories import CustomUserFactory
+import django
+from account.models import CustomUser
+class TestCustomUserValidators(django.test.TestCase):
 
+     def test_valid_user_creation(self):
+        user = CustomUserFactory()
+        self.assertEqual(CustomUser.objects.count(), 1)
+        self.assertEqual(user.is_active, False)
+    
+     def test_phone_valid_user_creation(self):
+        user = CustomUserFactory(phone = '5899298')
```

Comment:
The `=` sign does not have space on either side, when it is in a function call, indicating argument. Thus it should be, `phone='....'`

Reply:
Thank you for recommendation. I used black for formatting.